Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.cpp

```diff
@@ -68,6 +86,55 @@ namespace ROS2
         }
     }
 
+    AZ::Crc32 LidarSensorConfiguration::SegmentationClassesChangeNotify()
+    {
+        bool unknown_present = false;
```

Comment:
I would add a little more check on user input.
I can see that you can have multiple classes with the same ID that cause unexpected results.
I would rather split this component into game and editor components, and use Qt dialogs to warn the user on the wrong configuration.
Of course it should be done in next PRs.

Reply:
I can see that you can have multiple classes with the same ID that cause unexpected results. (In practice results, are predictable, as only the latest (in order displayed in editor) name/color for given id will be used